hack/test-unit.sh

```diff
@@ -26,3 +26,20 @@ do
     os::log::info "Testing ${PACKAGE}"
     go test $PACKAGE
 done
+
```

Comment:
So our `ci` jobs will call `make test-unit`. Does this need a `makefile` update? Thinking.

Reply:
nope, i see your `ocp` tag in the second run. great!